.github/workflows/rust.yml

```
@@ -55,7 +55,7 @@ jobs:
 
   coverage:
     name: Code Coverage
-    if: github.event_name != 'release'
+    if: github.event_name != 'release' && github.repository == 'maplibre/maplibre-tile-spec'
```

LOL, this is not what I meant :rofl: 

```suggestion
    if: github.event_name != 'release' && github.repository_owner == 'maplibre'
```